### ISSUE TITLE
Security/Screen Lock/Enhanced PIN privacy seems to override Privacy/Show passwords

### ISSUE THREAD

**Issue by sqrtiswap** [GrapheneOS/os-issue-tracker#2271]
Hi,

when disabling `Security/Screen Lock/Enhanced PIN privacy` not only does this highlight the keys on the lock screen while pressing but it also makes the last digit entered shortly visible.
Whether or not `Privacy/Show passwords` is enabled seems to be completely irrelevant: If `Security/Screen Lock/Enhanced PIN privacy` is enabled, then both highlighting and last digit is visible, if it is disabled, then neither is visible - tested in each case with `Privacy/Show passwords` enabled/disabled.

I would have expected two settings to provide more fine-grained control, instead one of them seems to be completely irrelevant.

**Comment by thestinger** [comment]
This is how this works now on Android. GrapheneOS doesn't change anything beyond the defaults.

**Comment by sqrtiswap** [comment]
Ok, I see. Should I then just close this issue since the fix should be made in AOSP?

**Comment by thestinger** [comment]
We could change it ourselves but it seems the people who worked on this intended it to work this way for some reason. We don't know why.

**Comment by sqrtiswap** [comment]
Ok, I also think that it used to be different. I believe I had it set up at some point to have the highlighting but not the visible last digit.
So I would be in favour of having it like that again but that's of course easy for me to say, I don't have to maintain this.

**Comment by thestinger** [comment]
It's a new feature in Android 13 QPR3, so yes, it changed recently. Nothing to do with GrapheneOS changes.

**Comment by sqrtiswap** [comment]
No, I understood that. I'm sorry for not making it clearer: I was commenting on "We could change it ourselves". That read like a slim chance you might think about changing this behaviour downstream back to how it used to be (discarding whatever upstream thought when they made their change) and I wanted to express that I would appreciate that. But I'm also already happy to now know what is happening. So, thanks for the explanation :)
